src/v2/view-builder/views/SSOExtensionView.js

```diff
@@ -33,4 +35,5 @@ const Body = BaseForm.extend({
 
 export default BaseAuthenticatorView.extend({
   Body,
+  Header: '',
```

Comment:
`Header: null`

Reply:
@haishengwu-okta this throws error: `View.js:347 Uncaught (in promise) TypeError: Cannot read property 'prototype' of null`